src/components/Company/Offers/Manage/CompanyOffersManagementWidget.js

```diff
@@ -0,0 +1,125 @@
+import { Card, CardContent, Typography } from "@material-ui/core";
+import { format, parseISO } from "date-fns";
+import React, { useState, useEffect } from "react";
+import { fetchCompanyOffers } from "../../../../services/companyOffersService";
+import ControlledSortableSelectableTable from "../../../../utils/Table/ControlledSortableSelectableTable";
+import FilterableTable from "../../../../utils/Table/FilterableTable";
+import { generateTableCellFromField } from "../../../../utils/Table/utils";
+import { columns } from "./CompanyOffersManagementSchema";
+import PropTypes from "prop-types";
+
+const CompanyOffersNonFullfilledRequest = ({ isLoading, error }) => {
+    if (isLoading) {
+        return (
+            <h1>Loading...</h1>
+        );
+    } else if (error) {
+        return (
+            <h1>
+                {error.message}
+            </h1>
+        );
+    } else {
+        return null;
+    }
+};
+
+const generateRow = ({ position, location }) => ({
+    fields: {
+        title: { value: position, align: "left" },
+        publishStartDate: { value: format(parseISO((new Date()).toISOString()), "yyyy-MM-dd") },
+        publishEndDate: { value: format(parseISO((new Date()).toISOString()), "yyyy-MM-dd") },
+        location: { value: location },
+    },
+    payload: {
+
+    },
+});
+
+const sorters = {
+
+};
+
+const filters = [
+
+];
+
+const RowActions = () => (
+    <Typography>Rows Actions</Typography>
+);
+
+
+const CompanyOffersManagementWidget = () => {
+    const [offers, setOffers] = useState(null);
+    const [isLoading, setIsLoading] = useState(true);
+    const [error, setError] = useState(null);
+
+    useEffect(() => {
+        fetchCompanyOffers().then((offers) => {
+            /* TODO: SHOULD NOT RUN WHEN COMPONENT IS UNMOUNTED: CANCEL PROMISE */
+            const fetchedRows = offers.reduce((rows, row) => {
+                rows[row.id] = generateRow(row);
+                return rows;
+            }, {});
+
+            setOffers(fetchedRows);
+            setIsLoading(false);
+        }).catch((err) => {
+            setError(err);
+            setIsLoading(false);
+        });
+    }, []);
+
+    const RowComponent = ({ rowKey, labelId }) => {
+        const fields = offers[rowKey].fields;
+
+        return (
+            <>
+                {Object.entries(fields).map(([fieldId, fieldOptions], i) => (
+                    generateTableCellFromField(i, fieldId, fieldOptions, labelId)
+                ))}
+            </>
+        );
+    };
+
+    RowComponent.propTypes = {
+        rowKey: PropTypes.string.isRequired,
+        labelId: PropTypes.string.isRequired,
+    };
+
+    return (
+        <div>
+            <Card>
```

Comment:
After the refactor from PageLayout, you should remove this Card, since it will already be rendered above